src/utils/directory.js

```diff
@@ -15,6 +15,13 @@
 const fs = require('fs');
 const path = require('path');
 
+const getDynamicFolderName = (folderNameTemplate, templateOptions) => {
```

Comment:
folderNameTemplate sounds like the template belongs to the folder rather than it being a directory from the template

consider changing to :
```
const getDynamicDirectoryName = (templateDirectoryName, templateOptions) => {
```